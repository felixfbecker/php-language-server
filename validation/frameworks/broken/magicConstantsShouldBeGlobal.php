<?php

namespace B;

echo __FILE__;