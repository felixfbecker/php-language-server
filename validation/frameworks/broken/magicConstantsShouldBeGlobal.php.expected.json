{
    "references": [],
    "definitions": {
        "B": {
            "extends": [],
            "isGlobal": true,
            "isStatic": false,
            "canBeInstantiated": false,
            "symbolInformation": {
                "name": "B",
                "kind": 3,
                "location": {
                    "uri": "\/Users\/roblou\/code\/php-language-server\/tests\/Validation\/..\/..\/validation\/frameworks\/broken\/magicConstantsShouldBeGlobal.php"
                },
                "containerName": ""
            },
            "type__class": "LanguageServer\\Tests\\ValidationTest",
            "type": null,
            "documentation": null
        }
    }
}